packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -310,6 +319,11 @@ class DebuggerController extends DisposableController
     return _scriptCache[scriptRef.id];
   }
 
+  ScriptRef getScriptRefFromUri(String uri) {
```

Comment:
dart style: avoid prefixing method names with the word `get`.

Consider keeping a cached map from uri to ScriptRef to avoid the O(num scripts) search each time the scriptRefFromUri is needed. Probably not a needed optimization but scanning through the full list of 30,000 scripts might be noticeable for large applications.

Reply:
Changed to `scriptRefForUri`.

I'm not sure it'll be a performance issue - this is currently only called (once) on an explicit user action. I'll cache the look up in map however - we might refactor the code in the future and end up calling this method more than expected.